### PR TITLE
Add permission for Flatpak.

### PR DESCRIPTION
**Pull request by istori1**
## Description

Adding `--talk-name=org.freedesktop.Flatpak` as this is needed to launch applications on the host. Example: `flatpak-spawn --host sleep 1m`.

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the docstring/documentation-blocks for new or existing methods/components

### CODE REVIEW

File: packaging/linux/flatpak/dev.lizardbyte.sunshine.yml [hunk 1]
```diff
@@ -5,14 +5,15 @@ sdk: org.freedesktop.Sdk
...
   - --socket=x11
   - --socket=wayland
   - --socket=pulseaudio
+  - --share=ipc
   - --share=network
   - --device=all
   - --persist=.config/sunshine
   - --system-talk-name=org.freedesktop.Avahi
+  - --talk-name=org.freedesktop.Flatpak
   - --env=PULSE_PROP_media.category=Manager
```
Comment (ReenigneArcher) [inline packaging/linux/flatpak/dev.lizardbyte.sunshine.yml:17]: Is the order of these args important? If not can we put them in alphabetical order?

**Review by ReenigneArcher — COMMENTED** [review]
Is any documentation update needed for this?

File: packaging/linux/flatpak/dev.lizardbyte.sunshine.yml (hunk 1 above)
Comment (istori1) [reply packaging/linux/flatpak/dev.lizardbyte.sunshine.yml:17]: It is not important. That sounds good to me.

**Comment by istori1** [comment]
> 

We should document using "flatpak-spawn --host app/game executable"

**Comment by ReenigneArcher** [comment]
> > 
> 
> We should document using "flatpak-spawn --host app/game executable"

Perhaps it should be defined in this section? https://github.com/istori1/Sunshine/blob/patch-2/docs/source/about/usage.rst#application-list

OR https://github.com/istori1/Sunshine/blob/patch-2/docs/source/troubleshooting/linux.rst#linux

**Comment by istori1** [comment]
> > > 
> > 
> > 
> > We should document using "flatpak-spawn --host app/game executable"
> 
> Perhaps it should be defined in this section? https://github.com/istori1/Sunshine/blob/patch-2/docs/source/about/usage.rst#application-list
> 
> OR https://github.com/istori1/Sunshine/blob/patch-2/docs/source/troubleshooting/linux.rst#linux

It should be in the usage section.

**Pull request merged**